apps/web/pages/bookings/[status].tsx

```diff
@@ -6,7 +6,7 @@ import { z } from "zod";
 
 import { WipeMyCalActionButton } from "@calcom/app-store/wipemycalother/components";
 import { getLayout } from "@calcom/features/MainLayout";
-import { FiltersContainer } from "@calcom/features/bookings/components/FiltersContainer";
+import { FiltersContainer, FilterToggleCta } from "@calcom/features/bookings/components/FiltersContainer";
```

Comment:
I think just `FiltersToggle` would be a better name for the new component. What do you think?

Reply:
I have changed the name of the component.